src/api/app/controllers/application_controller.rb

```diff
@@ -456,4 +457,13 @@ def forward_from_backend(path)
   def shutup_rails
     Rails.cache.silence! unless Rails.env.development?
   end
+
+  def set_influxdb_tags
+    anonymous = User.current.present? && !User.current.is_nobody?
+    InfluxDB::Rails.current.tags = {
+      beta: anonymous && User.current.in_beta?,
```

Comment:
what happen if `User.current` is `nil`? This would break, you need `User.current.try(:in_beta?)`

Reply:
anonymous is only true if there is a user (for whatever reason :)